File: resolve_conflicts.py
import re

def resolve_conflicts_keep_head(filepath):
    with open(filepath, 'r', encoding='utf-8') as f:
        content = f.read()
    
    pattern = r'<<<<<<< HEAD\n(.*?)\n=======\n(.*?)\n>>>>>>> origin/Krit_front'
    
    count = len(re.findall(pattern, content, re.DOTALL))
    
    resolved = re.sub(pattern, r'\1', content, flags=re.DOTALL)
    
    with open(filepath, 'w', encoding='utf-8') as f:
        f.write(resolved)
    
    print(f"Resolved {count} conflicts in {filepath}")

files = [
    r"d:\System_Analysis_and_Design\smart-moto-service-center\frontend\src\pages\mechanic\MechanicJobDetailPage.tsx",
    r"d:\System_Analysis_and_Design\smart-moto-service-center\frontend\src\pages\mechanic\MechanicHistoryPage.tsx",
    r"d:\System_Analysis_and_Design\smart-moto-service-center\frontend\src\pages\foreman\JobOrdersPage.tsx",
    r"d:\System_Analysis_and_Design\smart-moto-service-center\frontend\src\pages\foreman\JobHistoryPage.tsx",
    r"d:\System_Analysis_and_Design\smart-moto-service-center\frontend\src\pages\foreman\JobDetailPage.tsx",
]

for f in files:
    resolve_conflicts_keep_head(f)

print("Done!")

File: resolve_conflicts.js
const fs = require('fs');

const files = [
  'frontend/src/pages/mechanic/MechanicJobDetailPage.tsx',
  'frontend/src/pages/mechanic/MechanicHistoryPage.tsx',
  'frontend/src/pages/foreman/JobOrdersPage.tsx',
  'frontend/src/pages/foreman/JobHistoryPage.tsx',
  'frontend/src/pages/foreman/JobDetailPage.tsx',
];

files.forEach(f => {
  let c = fs.readFileSync(f, 'utf8');
  const re = /<<<<<<< HEAD\n([\s\S]*?)\n=======\n[\s\S]*?\n>>>>>>> origin\/Krit_front/g;
  const count = (c.match(re) || []).length;
  c = c.replace(re, '$1');
  fs.writeFileSync(f, c, 'utf8');
  console.log('Resolved ' + count + ' conflicts in ' + f);
});

console.log('Done!');
